chromium_src/chrome/installer/util/util_constants.cc

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/installer/util/util_constants.h"
+
+#define kGoogleUpdateIsMachineEnvVar kGoogleUpdateIsMachineEnvVar_ChromiumImpl
+#define kChromeExe kChromeExe_Unused
+
+#include "../../../../../chrome/installer/util/util_constants.cc"
+
+#undef kGoogleUpdateIsMachineEnvVar
+#undef kChromeExe
+
+namespace installer {
+
+namespace env_vars {
+
+#if defined(OFFICIAL_BUILD)
```

Comment:
I think we can safely delete this guard because we only support installer build for official release.
Using `kGoogleUpdateIsMachineEnvVar_ChromiumImpl` below doesn't have much meaning.

Reply:
I've talked to @simonhong offline to better understand this and we agreed on that removing that OFFICIAL_BUILD guard would make sense in that it's not actually used for the installer, but that would divert a bit from the conversion we're trying to do here, as that would slightly change the logic originally implemented in the patch we're removing, which is considering official and non-official builds.

Therefore, we agreed to keep this bit in this PR and then work separately on another PR not just to remove that guard from this override, but to also do it from other overrides and patches affecting //chrome/installer/** files in general, so that we can cleanup and simplify things further in a more consistent way.

Reply:
New issue filed to track down that simplification effort: https://github.com/brave/brave-browser/issues/14364